server/routes/termdb.cluster.ts

```diff
@@ -224,7 +218,10 @@ async function validateNative(q: GeneExpressionQueryNative, ds: any, genome: any
 			if (!g.chr) {
 				// quick fix: newly added gene from client will lack chr/start/stop
 				const re = getResultGene(genome, { input: g.gene, deep: 1 })
-				if (!re.gmlst || re.gmlst.length == 0) throw 'unknown gene'
+				if (!re.gmlst || re.gmlst.length == 0) {
+					console.warn('unknown gene:' + g.gene) //throw 'unknown gene'
```

Comment:
thanks for the fix; please comment a todo that unknown genes should be notified to client